pennylane/qchem/tapering.py

```diff
@@ -347,13 +358,13 @@ def taper(h, generators, paulixops, paulix_sector):
     eigenvalues is defined as the Pauli sector.
 
     Args:
-        h (Hamiltonian): PennyLane Hamiltonian
-        generators (list[Hamiltonian]): generators expressed as PennyLane Hamiltonians
+        h (~.Operator): Hamiltonian as a PennyLane operator
```

Comment:
Yes we now support both `Hamiltonian` and `Operator`. Since `Operator` is a subclass of `Hamiltonian` we can get away with just displaying type `Operator`